src/FeeComponents.js

```diff
@@ -0,0 +1,135 @@
+     * @param {Long} props.transactionVerification
+     * @param {Long} props.transactionRamByteHour
+     * @param {Long} props.transactionStorageByteHour
+     * @param {Long} props.contractTransactionGas
+     * @param {Long} props.transferVolumeHbar
+     * @param {Long} props.responseMemoryByte
+     * @param {Long} props.responseDiskByte
+     */
+    constructor(props) {
+        /*
+         * A minimum, the calculated fee must be greater than this value
+         *
+         * @type {Long}
+         */
+        this.min = props.min;
+        /*
+         * A maximum, the calculated fee must be less than this value
+         *
+         * @type {Long}
+         */
+        this.max = props.max;
+        /*
+         * A constant contribution to the fee
+         *
+         * @type {Long}
+         */
+        this.constant = props.constant;
+        /*
+         * The price of bandwidth consumed by a transaction, measured in bytes
+         *
+         * @type {Long}
+         */
+        this.transactionBandwidthByte = props.transactionBandwidthByte;
+        /*
+         * The price per signature verification for a transaction
+         *
+         * @type {Long}
+         */
+        this.transactionVerification = props.transactionVerification;
+        /*
+         * The price of RAM consumed by a transaction, measured in byte-hours
+         *
+         * @type {Long}
+         */
+        this.transactionRamByteHour = props.transactionRamByteHour;
+        /*
+         * The price of storage consumed by a transaction, measured in byte-hours
+         *
+         * @type {Long}
+         */
+        this.transactionStorageByteHour = props.transactionStorageByteHour;
+        /*
+         * The price of computation for a smart contract transaction, measured in gas
+         *
+         * @type {Long}
+         */
+        this.contractTransactionGas = props.contractTransactionGas;
+        /*
+         * The price per hbar transferred for a transfer
+         *
+         * @type {Long}
+         */
+        this.transferVolumeHbar = props.transferVolumeHbar;
+        /*
+         * The price of bandwidth for data retrieved from memory for a response, measured in bytes
+         *
+         * @type {Long}
+         */
+        this.responseMemoryByte = props.responseMemoryByte;
+        /*
+         * The price of bandwidth for data retrieved from disk for a response, measured in bytes
+         *
+         * @type {Long}
+         */
+        this.responseDiskByte = props.responseDiskByte;
+    }
+    /**
+     * @param {Uint8Array} bytes
+     * @returns {FeeComponents}
+     */
+    static fromBytes(bytes) {
+        return FeeComponents.fromProtobuf(proto.FeeComponents.decode(bytes));
+    }
+    /**
+     * @param {proto.IFeeComponents} feeComponents
+     * @returns {FeeComponents}
+     */
+    static fromProtobuf(feeComponents) {
+        return new FeeComponents({
+            min: /** @type {Long} */ (feeComponents.min),
```

Comment:
Once you change the constructor to allow for nullable fields you should remove the force casting here.